### ISSUE TITLE
Sheep (oaries aries): No ensembl gene homology 

### ISSUE THREAD

**Issue by HenrikCordes** [BgeeDB/BgeeDB_R#28]
Dear BgeeDB developers,

the sheep data from BgeeDB is annotated with ensembl gene ID from oaries aries which do not have (not jet) homology mappings to human or other species. I reached out to ensembl to ask if this is planed. Hovever the oarambouillet sheep genes do have the homology mapping. 

When I tried to use the (gene) Expression comparison from Bgee for sheep genes, this fail. I guess its the missing homology mappings for oaries aries. I guess currently, sheep expressions can not be compared to other species with Bgee?

Maybe ensembl will update the homologies or you could use the oarambouillet mapping in a future release.

Keep up the great work!

Best,
Henrik

**Comment by jwollbrett** [comment]
Hello Henrik,

Thank you for your feedback.
When you say that the expression comparison failed does it mean that you
had no result for your genes or did you get an error?
Could you please send me the list of ensembl gene IDs you provided to the
expression comparison tool?
We do have homologies of Ovis aries in our database. The source of our
homologies is OMA.

Best,

Julien

Le ven. 22 juil. 2022 à 09:47, Henrik Cordes ***@***.***> a
écrit :

**Comment by HenrikCordes** [comment]
Hi @jwollbrett,

Thank you for the quick reply. I belive I mixed up some things regarding the functionality of the expression tool. 
I had the impression I give a list of genes, from a specific species, and the tool would give me an organ resolved comparison of the gene list with other species. Later I did realize I would have to give all gene identifiers (for all species I would like to be compared to).

I do have issues mapping the Sheep genes to human, but that is a different story. Thank you for pointing me to the OMA source. 

Best,
Henrik
